### ISSUE TITLE
Localizations of Macros in Desk.com

### ISSUE THREAD

**Issue by stephaniemcv** [MozillaFoundation/Advocacy#401]
Detailed description of work: Submit current macro text to L10N to be localized in Desk.com

_Start date: October 1, 2016_ 
_Due date: October 17, 2016_ 
Note: The dates need to be exactly in the format indicated above for proper integration with the Gantt chart.

CC:

**Comment by valianttry** [comment]
cc @TheoChevalier

**Comment by Matt-Holland** [comment]
@roseregina  and @stephaniemcv  Is this still needed?

**Comment by stephaniemcv** [comment]
Yup. I believe  @WillatMozFdn is still refining some text before we hand over to @TheoChevalier.

**Comment by TheoChevalier** [comment]
Can you please share with me the working documents *before* handing off the strings to me?

**Comment by stephaniemcv** [comment]
Sorry Theo, we only do mind to mind communication here. ;-)

Shared the file with you.
